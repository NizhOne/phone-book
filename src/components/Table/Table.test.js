import React from 'react';
import TableComponent from './Table';
import renderer from 'react-test-renderer';


test('test', () => {
    const component = renderer.create(
        <TableComponent tableData={[]} editItem={() => true} deleteItem={() => true} />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});