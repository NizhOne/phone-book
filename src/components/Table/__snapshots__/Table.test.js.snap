// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test 1`] = `
<div
  className="table-wrapper"
>
  <div
    className="table-row"
  >
    <div
      className="table-row__determining"
    >
      <span
        className="table-row__determining_main"
      >
        Name
      </span>
      Â 
    </div>
    <div
      className="table-row__values"
    >
      <div
        className="table-row__values_value"
      >
        Phone number
      </div>
      <div
        className="table-row__values_value"
      >
        Email
      </div>
    </div>
  </div>
</div>
`;
